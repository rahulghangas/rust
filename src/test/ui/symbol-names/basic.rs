#![feature(rustc_attrs)]

#[rustc_symbol_name] //~ ERROR symbol-name(_RNaCs4fqI2P2rA04_5basic4main)
#[rustc_def_path] //~ ERROR def-path(main)
fn main() {
}
