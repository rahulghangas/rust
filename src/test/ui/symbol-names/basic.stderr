error: symbol-name(_RNaCs4fqI2P2rA04_5basic4main)
  --> $DIR/basic.rs:3:1
   |
LL | #[rustc_symbol_name] //~ ERROR symbol-name(_RNaCs4fqI2P2rA04_5basic4main)
   | ^^^^^^^^^^^^^^^^^^^^

error: def-path(main)
  --> $DIR/basic.rs:4:1
   |
LL | #[rustc_def_path] //~ ERROR def-path(main)
   | ^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

